test/unit/email_regex_test.rb

```diff
@@ -0,0 +1,127 @@
+# encoding: utf-8
+# rubocop:disable all
+require 'test_helper'
+
+class EmailRegexTest < ActiveSupport::TestCase
```

Comment:
Just an information: If you prefer RSpec we would gladly see RSpec tests in the future over Minitest.